Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -1189,6 +1189,8 @@ public extension Purchases {
         await self.paywallEventsManager?.track(paywallEvent: paywallEvent)
     }
 
+    #if CUSTOMER_CENTER_ENABLED
+
     /// Used by `RevenueCatUI` to download customer center data
     func loadCustomerCenter() async throws -> CustomerCenterConfigData {
```

Comment:
Ahh just noticed we're not wrapping the `CustomerCenterConfigData` into these flags right? We probably should to avoid exposing that API